vsintegration/src/FSharp.LanguageService.Base/DocumentTask.cs

```diff
@@ -182,807 +182,5 @@ internal static void DoOnUIThread(Action callback)
         }
     }
 
-    // DocumentTask is associated with an IVsTextLineMarker in a specified document and 
```

Comment:
Can we rename this file to `UIThread.cs` now that it only contains the UIThread class?